bin/MadGraph5_aMCatNLO/cards/production/13TeV/HPlusHF/HPlusBottom_4FS_MuRFScaleDynX0p50_M125_TuneCP5_13TeV_amcatnlo_pythia8/HPlusBottom_4FS_MuRFScaleDynX0p50_M125_TuneCP5_13TeV_amcatnlo_pythia8_run_card.dat

```diff
@@ -0,0 +1,170 @@
+#***********************************************************************
+  average = event_norm ! valid settings: average, sum, bias
+#***********************************************************************
+# Number of points per integration channel (ignored for aMC@NLO runs)  *
+#***********************************************************************
+  0.01 = req_acc_fo ! Required accuracy (-1=ignored, and use the 
+                    ! number of points and iter. below)
+# These numbers are ignored except if req_acc_FO is equal to -1
+  5000 = npoints_fo_grid ! number of points to setup grids
+  4 = niters_fo_grid ! number of iter. to setup grids
+  10000 = npoints_fo ! number of points to compute Xsec
+  6 = niters_fo ! number of iter. to compute Xsec
+#***********************************************************************
+# Random number seed                                                   *
+#***********************************************************************
+  0 = iseed ! rnd seed (0=assigned automatically=default)
+#***********************************************************************
+# Collider type and energy                                             *
+#***********************************************************************
+  1 = lpp1 ! beam 1 type (0 = no PDF)
+  1 = lpp2 ! beam 2 type (0 = no PDF)
+  6500.0 = ebeam1 ! beam 1 energy in GeV
+  6500.0 = ebeam2 ! beam 2 energy in GeV
+#***********************************************************************
+# PDF choice: this automatically fixes also alpha_s(MZ) and its evol.  *
+#***********************************************************************
+  'lhapdf'             = pdlabel  ! PDF set
+  $DEFAULT_PDF_SETS    = lhaid    ! if pdlabel=lhapdf, this is the lhapdf number
+  $DEFAULT_PDF_MEMBERS = reweight_PDF
+#***********************************************************************
+# Include the NLO Monte Carlo subtr. terms for the following parton    *
+# shower (HERWIG6 | HERWIGPP | PYTHIA6Q | PYTHIA6PT | PYTHIA8)         *
+# WARNING: PYTHIA6PT works only for processes without FSR!!!!          *
+#***********************************************************************
+  PYTHIA8  = parton_shower 
+  0.5      = shower_scale_factor ! multiply default shower starting
+                                 ! scale by this factor
+#***********************************************************************
+# Renormalization and factorization scales                             *
+# (Default functional form for the non-fixed scales is the sum of      *
+# the transverse masses divided by two of all final state particles    * 
+# and partons. This can be changed in SubProcesses/set_scales.f or via *
+# dynamical_scale_choice option)                                       *
+#***********************************************************************
+  False = fixed_ren_scale ! if .true. use fixed ren scale
+  False = fixed_fac_scale ! if .true. use fixed fac scale
```

Comment:
are you planing to use the fixed scale listed below? if yes, these two lines need to be true.

Reply:
No, we're planning to use dynamic scale, as stated in the name of the data cards, the value reported here for the fixed scale parameter was used during previous tests and left for completeness.